hbp_validation_framework/__init__.py

```diff
@@ -1459,8 +1459,8 @@ def register_model(self, app_id="", name="", alias=None, author="", organization
 
         Returns
         -------
-        UUID
-            (Verify!) UUID of the model description that has been created.
+        dict
```

Comment:
This should return the UUID, not a dict.

Reply:
Ok... will make the changes.
p.s. the above was only a change in the docs; the client needs to be updated to retrieve the UUID